iot/iot-service-base/pom.xml

```diff
@@ -46,6 +46,21 @@
       <version>${spring.boot.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-client-hotrod</artifactId>
+    </dependency>
```

Comment:
We shouldn't force this on all dependents. Could you mark them as "optional" here and then add them to the actual project.